environments/qa/modules/oaeqaautomation/manifests/init.pp

```diff
@@ -78,7 +79,7 @@
     $cron_log_file_path = "${log_file_dir}/`date +'\\%Y-\\%m-\\%d-\\%H-\\%M'`.log"
     cron { 'nightly-redeploy':
         ensure      => present,
-        command     => "${scripts_dir}/nightly.sh >> ${cron_log_file_path} 2>> ${cron_log_file_path}",
+        command     => "test -e /var/lock/redeploy || ${scripts_dir}/redeploy.sh >> ${cron_log_file_path} 2>> ${cron_log_file_path}",
```

Comment:
Should we add this test to the redeploy script as well?

I've accidentally wiped a QA server by just running `/opt/scripts/nightly.sh`. Maybe it can be overridden by running `/opt/scripts/nightly.sh --force` or something.